src/badges/tests/test_views.py

```diff
@@ -239,21 +239,21 @@ def setUp(self):
         self.badge_type = baker.make(BadgeType)
     
     def test_all_page_status_codes_for_anonymous(self):
-        ''' If not logged in then all views should redirect to home page or admin login '''
```

Comment:
Not redirecting to home page, redirecting to login page...

Reply:
fixed